Buddies/Onboarding/LoginVC.swift

```diff
@@ -60,15 +60,15 @@ class LoginVC: LoginBase {
             email: email,
             password: password,
             onError: { msg in self.showMessagePrompt(msg) },
-            onSuccess: { user in BuddiesStoryboard.Main.goTo() }
+            onSuccess: { user in self.performSegue(withIdentifier: "GetSignUpInfo", sender: self)}
```

Comment:
@JakeThurman is there some kind of storyboard Voodoo you had hooked up that did something other than segue?